employee/admin.py

```diff
@@ -1,16 +1,161 @@
 from django.contrib import admin
 from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import PermissionsMixin
+
 from .models import Employee, Designation
 
+from .forms import EmployeeCreationForm, EmployeeChangeForm
+
+
+# Remove fields from fieldset
+
+
+def remove_from_fieldsets(fieldsets, fields):
+    for fieldset in fieldsets:
+        for field in fields:
+            if field in fieldset[1]['fields']:
+                new_fields = []
+                for new_field in fieldset[1]['fields']:
+                    if not new_field in fields:
+                        new_fields.append(new_field)
+
+                fieldset[1]['fields'] = tuple(new_fields)
+                break
 
-class EmployeeAdmin(admin.ModelAdmin):
-    exclude = ("createdBy", "updatedBy")
-    list_display = ("picture_tag", "fullname", "join_date",  "designation")
-    list_filter = ("designation", "join_date")
+# Remove fieldsets
+
+
+def remove_fieldsets(fieldsets, fieldsetname):
+    new_fieldsets = []
+    for fieldset in fieldsets:
+        if fieldset[0] is not fieldsetname:
+            new_fieldsets.append(fieldset)
+    return new_fieldsets
```

Comment:
Let's use list comprehension and comment inside the function. Something like this.
```suggestion

def remove_fieldsets(fieldsets, fieldsetname):
    """
    Remove fieldset from fieldsets using name
    """
    return [
        fieldset for fieldset in fieldsets if fieldset[0] != fieldsetname
    ]
```

Reply:
Used list comprehension and comment inside the function. Not familiar with list comprehension. Is there any converter that does it ?

Reply:
> Is there any converter that does it?

Not sure what you meant by this.

This should make list comprehension clear https://www.programiz.com/python-programming/list-comprehension

Reply:
Done